### PR TITLE
New Tag property

### PR DESCRIPTION
**Pull request by Starwer**
As explain in #69, the following tags have been globally added: 
  - Subtitle
  - Description
  - PerformersRole
  - DateTagged
Also, to support #64, the following improvement has been implemented:
+ Format the Track as zero-filled two-digits in formats: Matroska, Id3v2, Ogg;

### CODE REVIEW

File: tests/fixtures/TagLib.Tests.FileFormats/StandardTests.cs [hunk 1]
```diff
@@ -84,48 +100,55 @@ public static void SetTags (Tag tag, TestTagLevel level = TestTagLevel.Normal)
...
-			Assert.AreEqual ("TEST copyright", tag.Copyright);
+			Assert.AreEqual("TEST copyright", tag.Copyright);
 			Assert.AreEqual (100, tag.Disc);
 			Assert.AreEqual (101, tag.DiscCount);
 			Assert.AreEqual ("TEST genre 1; TEST genre 2", tag.JoinedGenres);
 			Assert.AreEqual ("TEST grouping", tag.Grouping);
 			Assert.AreEqual ("TEST lyrics 1\r\nTEST lyrics 2", tag.Lyrics);
 			Assert.AreEqual ("TEST performer 1; TEST performer 2", tag.JoinedPerformers);
-			Assert.AreEqual ("TEST title", tag.Title);
+			Assert.AreEqual("TEST title", tag.Title);
```
Comment (decriptor) [inline tests/fixtures/TagLib.Tests.FileFormats/StandardTests.cs]: There should be a space between the method name and the opening `(`.

**Comment by decriptor** [comment]
Does this break the API? I think I saw a switch from `comment` to `Description`?

File: tests/fixtures/TagLib.Tests.FileFormats/StandardTests.cs (hunk 1 above)
Comment (Starwer) [reply tests/fixtures/TagLib.Tests.FileFormats/StandardTests.cs]: Ok, I'll fix that.

File: src/TagLib/Ogg/XiphComment.cs [hunk 2]
```diff
@@ -656,22 +737,13 @@ IEnumerator IEnumerable.GetEnumerator()
 		///    langword="null" /> if no value is present.
 		/// </value>
 		/// <remarks>
-		///    This property is implemented using the "COMMENT" or
-		///    "DESCRIPTION" field, preferring "DESCRIPTION" but using
-		///    "COMMENT" if that is the field used by the comment.
+		///    This property is implemented using the "COMMENT" field.
 		/// </remarks>
 		public override string Comment {
-			get {
```
Comment (Starwer) [inline src/TagLib/Ogg/XiphComment.cs]: @decriptor : There is indeed a slight API change. This happens in the Xiph implementation. 
Before, the "COMMENT" and "DESCRIPTION" were both mapped to the TagLib field `Comment`. 
Now, this obviously doesn't apply anymore as we have a separated ``Comment` and ``Description` filed. 
As a result, there is a change in behaviour, compare to previous implementations. 
However, I think this PR makes it more standard-compliant and more clear on what to expect from Comment/Description.

Comment (decriptor) [reply src/TagLib/Ogg/XiphComment.cs]: I'm good with that. Lets just make sure we document that in the release notes. We should probably look at getting release out before the end of the year :)

Comment (Starwer) [reply src/TagLib/Ogg/XiphComment.cs]: I think that a release for Xmas would please many people indeed ;)

**Comment by rboy1** [comment]
If we are adding new tags for supporting video (which I really like), here are some of the most commonly used tags used by video players and DVR systems across the world today:

1. Title (present)
2. Subtitle
3. Genre (multiple/array) (present?)
4. Network or Channel (present?)
5. Description (present)
6. Credits (multiple/array) (present?)
7. Season (number)
8. Episode (number)
9. String to indicate type (movie, tv series, sports, news, documentary etc)
10. Recorded Date and Time (UTC format?)
11. Broadcast Date and Time (UTC format?)
12. Premiere Date and Time  (UTC format?)

Each platform has it's own specific tags like SageTV, nPVR, TiVO, HDHR, IceTV, Media Portal, WMC, Kodi etc but this was the list of the most common parameters I would gather between the platforms.

@Starwer  what do you think?

File: src/TagLib/Ogg/XiphComment.cs (hunk 2 above)
Comment (Starwer) [reply src/TagLib/Ogg/XiphComment.cs]: Fixed !

**Comment by Starwer** [comment]
@rboy1 : Interesting discussion that I wanted to initiate in issue: #69. Unless you think this is blocking the integration of the current PR, could you please copy/paste it in #69 ? I'll be more than pleased to continue the discussion there. 
Currently, I think the current implementation + this PR covers points: 1, 2, 3, 5, 7, 8. Further extension may be done in other PRs.
@decriptor : I've fixed the cosmetics issues reported and even extended the CheckNoTags coverage by checking more fields. Is there still anything blocking on this PR ?

File: tests/fixtures/TagLib.Tests.FileFormats/StandardTests.cs (hunk 1 above)
Comment (Starwer) [reply tests/fixtures/TagLib.Tests.FileFormats/StandardTests.cs]: Fixed !

File: src/TagLib/Ogg/XiphComment.cs (hunk 2 above)
Comment (Starwer) [reply src/TagLib/Ogg/XiphComment.cs]: Who is taking care of the release notes ? 
Shall I start updating the "NEWS" file with the user-visible changes I've done ?

Comment (decriptor) [reply src/TagLib/Ogg/XiphComment.cs]: @Starwer if you'd like to do that I'd very much appreciate it

**Comment by decriptor** [comment]
@Starwer @rboy1 that sounds good to me. I'll go ahead and merge this and let the conversation continue in that issue

**Pull request merged**